### PR TITLE
Implement Why Do We Ask For SSN Screen

### PR DESCRIPTION
**Pull request by spokenbird**

### CODE REVIEW

File: src/main/resources/templates/mdBenefitsFlow/applicantApplying.html [hunk 1]
```diff
@@ -22,9 +22,9 @@
                               content=~{::inputContent})}">
                 <th:block th:ref="inputContent">
                   <th:block
-                      th:replace="~{fragments/inputs/radio :: radio(inputName='isApplicantApplying',value='true', label=#{applicant-applying.yes})}"/>
+                      th:replace="~{fragments/inputs/radio :: radio(inputName='isApplicantApplying',value='Yes', label=#{applicant-applying.yes})}"/>
```
Comment (spokenbird) [inline src/main/resources/templates/mdBenefitsFlow/applicantApplying.html:25]: This fixes an issue with the mapping from this ticket: https://www.pivotaltracker.com/story/show/187040206

File: src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java [hunk 2]
```diff
@@ -34,7 +34,7 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
         results.put("applicantDOB", new SingleField("applicantDOB", (String) dob, null));
 
         results.put("applicantSSN",
-                new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("encryptedSSN")), null));
+                new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("applicantSSN")), null));
```
Comment (spokenbird) [inline src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java:37]: Fixes the same issue from this ticket:
https://www.pivotaltracker.com/story/show/187040206

Encryption will be handled later in this ticket:
https://www.pivotaltracker.com/n/projects/2688497/stories/186997138

File: src/main/resources/messages.properties [hunk 3]
```diff
@@ -448,6 +448,17 @@ applicant-applying.ssn=What is your Social Security Number (SSN)?
...
+# Why do we ask for SSNs
+ssn-why.title=Why we ask for Social Security Numbers (SSNs)?
+ssn-why.who=Who needs to give their SSN?
+ssn-why.who-1=Only add SSNs for people in your household who want to get benefits.
+ssn-why.who-2=You don't need to add SSNs for people who do not want to apply for benefits.
+ssn-why.why=Why do I need a SSN to get benefits?
+ssn-why.why-1=The law says everyone who gets SNAP food benefits needs to give their SSN.
+ssn-why.why-2=If you or someone else wants to get benefits without a SSN, it needs to be a good reason (such as being a newborn).
+ssn-why.why-3=The name of the law that gives us permission to ask for SSNs is the Food and Nutrition Act of 2008, as amended, 7 U.S.C. 2011?2036.
+ssn-why.what=What will you do with my SSN after I add it
```
Comment (spokenbird) [inline src/main/resources/messages.properties:460]: Noting this last section is not done yet, as it's pending active conversation happening here:
https://cfastaff.slack.com/archives/C06E3AQ76FK/p1708548463969789

Shalaka said we can submit as is and add the copy needed in a follow up ticket later.

**Review by analoo — APPROVED** [review]
LGTM

**Pull request merged**